scripts/e2eTestsOnVsix/extension_version.sh

```diff
@@ -2,12 +2,12 @@
 
 set -eu
 
-
-
-
 EXTENSION_INSIDER_VERSION=$(./node_modules/.bin/vsce show Prisma.prisma-insider --json | jq ".versions[0].version" | tr -d '"')
 EXTENSION_STABLE_VERSION=$(./node_modules/.bin/vsce show Prisma.prisma --json | jq ".versions[0].version" | tr -d '"')
 
+[ -z "$EXTENSION_INSIDER_VERSION" ] && echo "EXTENSION_INSIDER_VERSION is undefined, vsce show command failed, see logs." && exit 1
+[ -z "$EXTENSION_STABLE_VERSION" ] && echo "EXTENSION_STABLE_VERSION is undefined, vsce show command failed, see logs." && exit 1
```

Comment:
If one has to chain more then a single command conditionally, I'd usually go for a standard `if` block, but I don't have a strong opinion on this :)